src/parse.ts

```diff
@@ -184,6 +184,14 @@ export function parseSchema(
   rootSchema?: boolean
 ): TypeContent | undefined {
   function parseHelper(): TypeContent | undefined {
+    // @ts-ignore: Need to be able to check if honorCastTo contains any string
+    // and typescript is requiring the arg to only be 'number' or 'string'
+    if (settings.honorCastTo.includes(details.flags?.cast)) {
```

Comment:
I'm definitely not in love with this - so I'm all ears for a more graceful approach, although I'm not sure how else to handle with strict mode enabled. Seems like a miss as far as the transpiler goes to require that the includes type matches the array's contents.

Reply:
```suggestion
    if (details.flags?.cast && settings.honorCastTo.includes(details.flags?.cast)) {
```

Reply:
Awesome - I generally try to not use `as`  but I think there's no other way around it here.

I made this change, but also added a falsey check first although maybe not super necessary

Reply:
I often go overboard with falsey checks just to make sure